packages/select/index.demo.tsx

```diff
@@ -0,0 +1,51 @@
+import { h, Component } from 'skatejs';
+import { customElement } from '../_helpers';
+import Select from './Select';
```

Comment:
once you register them correctly you will need to import from index
`import {Select,Options} from './index'`